### PR TITLE
utils_test: Fix RemoteVMManager

### PR DESCRIPTION
**Pull request by dzhengfy**
1. Fix checking rsa private and public key on remote host instead of
local host because it is to execute the logon guest on remote host
instead of local host

2. Add some ssh option to avoid host key checking
3. Return the command execution result if required
4. Enable to avoid raising an exception when command execution occurs an
error
5. Fix syntax error

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: virttest/utils_test/__init__.py [hunk 1]
```diff
@@ -2004,15 +2011,32 @@ def check_network(self, vm_ip, count=5, timeout=60):
...
+            will return CmdResult object. False, raise an exception.
+
+        :raise: exceptions.TestFail, if the command fails
+        :return: CmdResult object
         """
-        cmd = "ssh %s@%s '%s'" % (vm_user, vm_ip, command)
+        ssh_options = "%s %s" % ("-o UserKnownHostsFile=/dev/null",
+                                 "-o StrictHostKeyChecking=no")
+        cmd = 'ssh %s %s@%s "%s"' % (ssh_options, vm_user, vm_ip, command)
+        cmdres = None
```
Comment (waynesun09) [inline virttest/utils_test/__init__.py]: s/cmdres/ret/g

Comment (dzhengfy) [inline virttest/utils_test/__init__.py]: @waynesun09 , updated and please help checking.

**Comment by waynesun09** [comment]
ACK

**Comment by dzhengfy** [comment]
@will-Do , could you help review too?

**Comment by will-Do** [comment]
ACK

**Pull request merged**